panic: runtime error: invalid memory address or nil pointer dereference

hi. 
i'm using prometheus-am-executor and i have a question.
i executed reboot script that default file in  ./prometheus-am-executor and i got runtime error.

how can i fix this error?
Should i modify go file?
[root@controller1 prometheus-am-executor]# ./prometheus-am-executor -v examples/reboot 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x8c12fc]

goroutine 1 [running]:
main.readCli(0x203000, 0x680, 0xc000235c00)
	/home/prometheus-am-executor/config.go:100 +0x29c
main.readConfig(0x98d0a0, 0x30, 0x30)
	/home/prometheus-am-executor/config.go:117 +0x26
main.main()
	/home/prometheus-am-executor/main.go:36 +0x42


thanks for your help.

--- a/config.go
+++ b/config.go
@@ -96,15 +96,17 @@ func readCli() (*Config, error) {
 		}
 	}
 
-	// Check that the commands specify resolved_signal values that we can parse
-	for i, cmd := range file.Commands {
-		_, err := cmd.ParseSignal()
-		if err != nil {
-			return nil, fmt.Errorf("Invalid resolved_signal specified for command %q at index %d: %w", cmd, i, err)
-		}
+	if file != nil {
+		// Check that the commands specify resolved_signal values that we can parse
+		for i, cmd := range file.Commands {
+			_, err := cmd.ParseSignal()
+			if err != nil {
+				return nil, fmt.Errorf("Invalid resolved_signal specified for command %q at index %d: %w", cmd, i, err)
+			}
 
-		if cmd.IgnoreResolved != nil && *cmd.IgnoreResolved {
-			log.Printf("Warning: command %q at index %d specifies a resolved_signal, and also specifies to ignore resolved alert. The signal won't be used.", cmd, i)
+			if cmd.IgnoreResolved != nil && *cmd.IgnoreResolved {
+				log.Printf("Warning: command %q at index %d specifies a resolved_signal, and also specifies to ignore resolved alert. The signal won't be used.", cmd, i)
+			}
 		}
 	}
 

